src/pkg/packager/create.go

```
@@ -313,45 +313,51 @@ func (p *Packager) addComponent(component types.ZarfComponent) (*types.Component
 			}
 
 			for idx, path := range chart.ValuesFiles {
-				chartValueName := helm.StandardName(componentPath.Values, chart) + "-" + strconv.Itoa(idx)
-				if err := utils.CreatePathAndCopy(path, chartValueName); err != nil {
-					return nil, fmt.Errorf("unable to copy chart values file %s: %w", path, err)
+				dst := helm.StandardName(componentPath.Values, chart) + "-" + strconv.Itoa(idx)
+				if utils.IsURL(path) {
+					utils.DownloadToFile(path, dst, component.CosignKeyPath)
```

errors are ignored here.